src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -79,9 +83,53 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
          _logger.AddInfo($"Qualification scenario finished in {timeSpent.ToDisplay()}");
       }
 
-      private Task<SaticFiles> copyStaticFiles(dynamic qualificationPlan)
+      private Task updateProjectsFullPath(IReadOnlyList<Project> projects) => Task.WhenAll(projects.Select(updateProjectFullPath));
+
+      private async Task<string> downloadRemoteFile(string url, string subFolder, string id, string type)
       {
-         var staticFiles = new SaticFiles();
+         _logger.AddDebug($"Downloading {type.ToLower()} file for {id}...");
+         var downloadFolder = Path.Combine(_runOptions.TempFolder, subFolder);
+         DirectoryHelper.CreateDirectory(downloadFolder);
+
+         
+         using (var wc = new WebClient())
+         {
+            try
+            {
+               var fileName = new Uri(url).Segments.Last();

```

Comment:
This will probably fail if the url is not formatted properly. However in the try catch, we should see an error anyways